### DISCUSSION TITLE
Question about Academic Discounts

### DISCUSSION THREAD

**Discussion by Cheng-Hz** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#791] [Q&A] [answered]
Hello,
I like AlDente very much and want to purchase AlDente Pro lifetime license. And I wonder if it has some academic discounts for college students. Thanks a lot.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @Cheng-Hz,

Unfortunately, we do not offer any kind of academic/student discount.

Cheers,
Matthias

**Reply by Cheng-Hz** [reply]
Hi @MatthiasKerbl,
Thank you for your answer, and I purchased AlDente Pro lifetime license. Hope AlDente could be better in the future.
Cheng

**Reply by MatthiasKerbl** [reply]
Thank you for your support, what could be improved?

**Reply by Cheng-Hz** [reply]
I suppose maybe it could add some features like setting up alerts when the charging plug is plugged in or unplugged. And under the status of unplugged, it will remind users to charge when the computer battery level is lower than the set value. 🤔

**Reply by MatthiasKerbl** [reply]
An alert, when the battery percentage is under a certain level when the MacBook is unplugged is something we already considered and might be implemented in one of the future updates.

**Reply by Cheng-Hz** [reply]
That's great.😃
